crates/oxc_sourcemap/tests/fixtures/esbuild/visualizer.snap

```diff
@@ -0,0 +1,208 @@
+---
+source: crates/oxc_sourcemap/tests/main.rs
+input_file: crates/oxc_sourcemap/tests/fixtures/esbuild/example.js
+snapshot_kind: text
+---
+- index.tsx
+(0:0) "import { " --> (1:0) "import { "
+(0:9) "h, " --> (1:9) "h as "
+(0:9) "h, " --> (1:14) "u, "
+(0:12) "Fragment, " --> (1:17) "Fragment as "
+(0:12) "Fragment, " --> (1:29) "l, "
+(0:22) "render } from " --> (1:32) "render as "
+(0:22) "render } from " --> (1:42) "c } from "
+(0:36) "'preact'\n" --> (1:51) "\"preact\";\n"
+- counter.tsx
+(0:0) "import { " --> (4:0) "import { "
+(0:9) "h, " --> (4:9) "h as "
+(0:9) "h, " --> (4:14) "t, "
+(0:12) "Component } from " --> (4:17) "Component as "
+(0:12) "Component } from " --> (4:30) "i } from "
+(0:29) "'preact'\n" --> (4:39) "\"preact\";\n"
+(1:0) "import { " --> (5:0) "import { "
+(1:9) "useState } from " --> (5:9) "useState as "
+(1:9) "useState } from " --> (5:21) "a } from "
+(1:25) "'preact/hooks'\n" --> (5:30) "\"preact/hooks\";\n"
+(12:7) "class " --> (6:0) "var "
+(12:13) "CounterClass extends " --> (6:4) "n = "
+(12:7) "class CounterClass extends " --> (6:8) "class extends "
```

Comment:
This looks tricky and I couldn't find a way to make it work like evanw.

![image](https://github.com/user-attachments/assets/edeeceeb-9f95-490e-beac-7a92a3734b72)